charts/nri-prometheus/values.yaml

```diff
@@ -220,41 +194,51 @@ config:
-# Pod scheduling priority
-# Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
-# priorityClassName: high-priority
-
+# -- Annotations to be added to all pods created by the integration.
+podAnnotations: {}
+
+
+# These values are managed by the common-library
+# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`
+cluster: ""
+# -- This set this license key to use. Can be configured also with `global.licenseKey`
+licenseKey: ""
+# -- In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there. Can be configured also with `global.customSecretName`
+customSecretName: ""
+# -- In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located. Can be configured also with `global.customSecretLicenseKey`
+customSecretLicenseKey: ""
+# -- Additional labels for chart pods. Can be configured also with `global.podLabels`
+podLabels: {}
+# -- Additional labels for chart objects. Can be configured also with `global.labels`
+labels: {}
+# -- Sets pod's priorityClassName. Can be configured also with `global.priorityClassName`
+priorityClassName: ""
+# -- Sets pod's hostNetwork. Can be configured also with `global.hostNetwork`
+hostNetwork: false
+# -- Sets pod's dnsConfig. Can be configured also with `global.dnsConfig`
+dnsConfig: {}
+# -- Sets security context (at pod level). Can be configured also with `global.podSecurityContext`
+podSecurityContext: {}
+# -- Sets security context (at container level). Can be configured also with `global.containerSecurityContext`
+containerSecurityContext: {}
+# -- Sets pod/node affinities. Can be configured also with `global.affinity`
+affinity: {}
+# -- Sets pod's node selector. Can be configured also with `global.nodeSelector`
 nodeSelector: {}
-
+# -- Sets pod's tolerations to node taints. Can be configured also with `global.tolerations`
 tolerations: []
-
-affinity: {}
-
-# Sends data to staging, can be set as a global.
-# global.nrStaging
-nrStaging: false
-
-# If enabled FedRamp Metrics API endpoint will be used.
-# https://docs.newrelic.com/docs/security/security-privacy/compliance/fedramp-compliant-endpoints#metric-api
 fedramp:
-  enabled: false
-
-# Proxy to be used by the emitters when submitting metrics. It should be
-# in the format [scheme]://[domain]:[port].
-# The emitter is the component in charge of sending the scraped metrics.
-# This proxy won't be used when scraping metrics from the targets.
-# By default it's empty, meaning that no proxy will be used.
-proxy:
-
-# Whether the integration should run in verbose mode or not.
-# Default: false
+  # fedramp.enabled -- (bool) Enables FedRAMP. Can be configured also with `global.fedramp.enabled`
+  # @default -- false
+  enabled:
+# -- (bool) Reduces number of metrics sent in order to reduce costs. Can be configured also with `global.lowDataMode`
```

Comment:
would make sense to ask the user to check the readme for further info? Or copy them directly here?

Reply:
Thinking about it, maybe it is enough to have them there, it is called readme for a reason 😅

Reply:
I cannot duplicate all the documentation 😅